ms2rescore/utils.py

```diff
@@ -66,11 +66,9 @@ def infer_spectrum_path(
             )
 
     # Match with file extension if not in resolved_path yet
-    if not _is_minitdf(resolved_path) and not re.match(
```

Comment:
todo: Can the function `_is_minitdf` be removed as well?

Reply:
`_is_minitdf` is still used in `infer_spectrum_path`.